rubygem-hammer_cli_katello/rubygem-hammer_cli_katello.spec

```diff
@@ -28,8 +28,8 @@ Requires:  %{?scl_prefix_ruby}ruby(release)
 Requires:  %{?scl_prefix_ruby}ruby(abi)
 %endif
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
-Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 0.1.3
-Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) < 0.5.0
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) >= 0.5.0
+Requires: %{?scl_prefix}rubygem(hammer_cli_foreman) < 1.0.0
```

Comment:
I could be missing something, but it looks like your greater than and less than are switched.